### DISCUSSION TITLE
no 'database drivers' command

### DISCUSSION THREAD

**Discussion by noreentry** [TPC-Council/HammerDB#151] [Q&A] [answered]
from https://www.hammerdb.com/docs/ch01s06.html#d0e389

> The command database drivers will show the installed ODBC Driver.

but i got `Error: invalid command name "database"`

**Comment by sm-shaw** [comment, marked as answer]
This is an OS command installed with the SQL Server on Linux ODBC packages. If you follow the instructions here you will have the database drivers command that you can run at the OS prompt or at the hammerdb prompt. 
https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server?view=sql-server-ver15

**Comment by sm-shaw** [comment]
Closed no further update
